ncm-network/src/main/perl/nmstate.pm

```diff
@@ -231,6 +231,34 @@ sub nmstate_file_dump
     }
 }
 
+# given ip/netmask address return cidr mask length
+sub get_masklen {
+    my ($self, $ip_netmask) = @_;
+    $self->verbose("Converting $ip_netmask to cidr notation");
```

Comment:
Is this relevant to an SA or just the programmer? Feels like debug would be more appropriate here

Reply:
yes agree. updated.